runtime/runtime-core/common/test/aws/smithy/kotlin/runtime/content/BigIntegerTest.kt

```diff
@@ -20,4 +20,9 @@ class BigIntegerTest {
     fun testBadBigInteger() {
         assertFails { BigInteger("1234567890foo1234567890") }
     }
+
+    @Test
+    fun testEquals() {
+        assertEquals(BigInteger("340282366920938463463374607431768211456"), BigInteger("340282366920938463463374607431768211456"))
```

Comment:
style/nit: it would be nice to have a single format for these new tests you added for `BigInteger` and `BigDecimal`
```kotlin
  val value = "0.340282366920938463463374607431768211456"
  assertEquals(BigDecimal(value), BigDecimal(value))
```